build.gradle

```diff
@@ -343,6 +355,14 @@ subprojects { subproject ->
 
     implementation group: 'commons-collections', name: 'commons-collections', version: '3.2.2'
     implementation group: 'commons-io', name: 'commons-io', version: '2.11.0'
+    implementation (group: 'com.microsoft.azure', name: 'azure-servicebus-jms-spring-boot-starter', version: '2.3.5'){
+      exclude group: 'org.simpleframework', module: 'simple-xml'
+      exclude group: 'org.nanohttpd', module: 'nanohttpd'
+    }
+
+    implementation group: 'org.springframework.boot', name: 'spring-boot-starter-activemq'
+    implementation group: 'org.apache.activemq', name: 'activemq-client', version: '5.16.1'
```

Comment:
Make repeated version constants a variable for simple one point of update.

Reply:
done